spec/insulate-expose_spec.lua

```diff
@@ -34,6 +34,32 @@ describe('Tests insulation', function()
   end)
 end)
 
+insulate('Tests insulation with values set to nil', function()
+  _G.some_global = true
+
```

Comment:
can we add a comment here that the tests in this block need to run in order (which generally is bad test design), because they test intermediate test state between tests (and hence there is no alternative).

Reply:
Ah, good point! Yepp, I'll add that.